minigalaxy/installer.py

```diff
@@ -182,3 +214,5 @@ def uninstall_game(game):
     shutil.rmtree(game.install_dir, ignore_errors=True)
     if os.path.isfile(game.status_file_path):
         os.remove(game.status_file_path)
+    if os.path.isfile(os.path.join(APPLICATIONS_DIR, game.name+".desktop")):
+        os.remove(os.path.join(APPLICATIONS_DIR, game.name+".desktop"))
```

Comment:
The same cosmetic request in both lines as above. ("{}.desktop."format(game.name))

Reply:
Done in all places within the file.